internal/schemautil/userconfig/stateupgrader/v0/serviceintegration/service_integration.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/aiven/terraform-provider-aiven/internal/schemautil/userconfig/stateupgrader/v0/dist"
 )
 
-const serviceIntegrationEndpointRegExp = "^[a-zA-Z0-9_-]*\\/{1}[a-zA-Z0-9_-]*$"
+var serviceIntegrationEndpointRegExp = regexp.MustCompile(`^[a-zA-Z0-9_-]*/[a-zA-Z0-9_-]*$`)
```

Comment:
This matches even to a slash `/` is that expected?

Reply:
it is